src/openrct2/object/Object.cpp

```diff
@@ -322,6 +322,29 @@ std::unique_ptr<IStream> ObjectAsset::GetStream() const
     return {};
 }
 
+std::string VersionString(Version version)
+{
+    return String::Format("%i.%i.%i", std::get<0>(version), std::get<1>(version), std::get<2>(version));
+}
+
+Version VersionTuple(std::string version)
```

Comment:
```suggestion
Version VersionTuple(const std::string& version)
```

Reply:
`std::string_view` is preferred over const references.